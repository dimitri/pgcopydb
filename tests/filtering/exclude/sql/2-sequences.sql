select last_value from seq.default_table_id_seq;
select last_value from seq.identity_table_id_seq;
select last_value from seq.standalone_id_seq;
select last_value from seq.standalone_smallint_id_seq;
select last_value from seq.standalone_minvalue_id_seq;
