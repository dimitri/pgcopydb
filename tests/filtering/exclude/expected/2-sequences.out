-[ RECORD 1 ]---
last_value | 667

-[ RECORD 1 ]---
last_value | 668

-[ RECORD 1 ]---
last_value | 669

-[ RECORD 1 ]---
last_value | 670

-[ RECORD 1 ]---
last_value | 671

